whitepapers/noise/noise.tex

```diff
@@ -55,22 +40,21 @@ \section{Preliminaries}
 \end{definition}
 If $\delta = 0$, we call this \emph{Pure DP}. If $\delta > 0$, we call this \emph{Approximate DP}.
 Note that, in practice, differential privacy could be thought of a bit more broadly -- as a bounded
-distance between joint distributions over the mechanism output and runtime.\footnote{Conceivably, this idea
-could be extended further to talk about distributions over all quantities related in any way to the underlying data.
-For example, imagine that the US government uses $\epsilon = 1$ if the President is in the data and $\epsilon = 10$
+distance between joint distributions over all entities that could be observable to an adversary.\footnote{For example, imagine that the 
```

Comment:
I feel like this footnote doesn't directly tie to the proceeding sentence? Maybe instead of "for example" in footnote could say "Note that DP does not avoid all attacks" or something?